.vib/mediawiki/goss/mediawiki.yaml

```diff
@@ -0,0 +1,50 @@
+group:
+  daemon:
+    exists: true
+user:
+  daemon:
+    exists: true
+file:
+  /opt/bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/images:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/cache:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    filetype: file
+    contains:
+      - "memory_limit = 256M"
+  /opt/bitnami/apache/conf/vhosts/mediawiki-vhost.conf:
+    exists: true
+    filetype: file
+    contains:
+      - "RewriteEngine On"
+      - "RewriteRule ^/?wiki(/.*)?$ %{DOCUMENT_ROOT}/index.php [L]"
+  /opt/bitnami/mediawiki/extensions/Scribunto:
+    exists: false
+command:
+  check-php-extensions:
+    exec: pear list
+    exit-status: 0
+    stdout:
+      - "Net_SMTP"
```

Comment:
Thank you for implementing the proposed test,

Is there a reason to omit the "Mail" module?

Reply:
Great catch! Thanks.